tpcds_variables.sh

```diff
@@ -10,6 +10,11 @@ export PGPORT="5432"
 # Add additional PostgreSQL refer:
 # https://www.postgresql.org/docs/current/libpq-envars.html
 
+# when OS configured with pre-login messages causing string pollution in
+# command output value; we should use ssh quite mode is we are expecting
+# command output. Additional flags can be added
+export ADDITION_SSH_OPTION="-q"
```

Comment:
Seems mismatch in the export variable name and usage

```suggestion
export ADDITION_SSH_OPTIONS="-q"
```

Reply:
thanks @deenav